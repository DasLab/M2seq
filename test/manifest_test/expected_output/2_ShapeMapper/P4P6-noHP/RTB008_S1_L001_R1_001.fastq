/usr/local/src/M2seq/test/manifest_test/output/1_Demultiplex/GGTATATGTACA/read1.fastq