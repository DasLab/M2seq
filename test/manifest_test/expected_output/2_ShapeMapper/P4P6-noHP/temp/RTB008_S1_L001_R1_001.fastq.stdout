Starting filtering
    Filtered /usr/local/src/M2seq/test/manifest_test/output/2_ShapeMapper/P4P6-noHP/RTB008_S1_L001_R1_001.fastq and wrote output to /usr/local/src/M2seq/test/manifest_test/output/2_ShapeMapper/P4P6-noHP/output/trimmed_reads/RTB008_S1_L001_R1_001.fastq
Filtering complete
